test/test-functions

```diff
@@ -96,8 +97,12 @@ $KERNEL_APPEND \
 }
 
 run_nspawn() {
+    local _nspawn_cmd="../../systemd-nspawn --register=no --kill-signal=SIGKILL --directory=$TESTDIR/nspawn-root $ROOTLIBDIR/systemd $KERNEL_APPEND"
```

Comment:
> sysv_generator = os.path.join(os.environ.get('builddir', '.'), 'systemd-sysv-generator')

@keszybz , actually I don't understand. How do you use this `builddir` variable?
Do you run something like `make check builddir=some-dir`?

The `builddir` was introduced by e28aa588f04
@martinpitt , why do we need this? `make check` doesn't pass the `builddir` to `test/sysv-generator-test.py`

`make check` passes the `srcdir` only:
> Test programs that need data files should look for them in srcdir (which is both a make variable and an environment variable made available to the tests), so that they work when building in a separate directory (see Build Directories in The Autoconf Manual), and in particular for the distcheck rule (see Checking the Distribution).

https://www.gnu.org/software/automake/manual/automake.html#Scripts_002dbased-Testsuites